doc/contributing/maintaining-V8.md

```diff
@@ -208,6 +208,62 @@ backport the fix:
 Abandoned V8 branches are supported in the Node.js repository. The fix needs
 to be cherry-picked in the Node.js repository and V8-CI must test the change.
 
+As an example for how to backport changes, consider the bug [RegExp show
+inconsistent result with other browsers](https://crbug.com/v8/5199). From the
+bug we can see that it was merged by V8 into 5.2 and 5.3, and not into V8 5.1
+(since it was already abandoned). Since Node.js `v6.x` uses V8 5.1, the fix
+needed to be backported.
+
+#### Backporting with `git-node` (recommended)
+
+You can use [`git-node`][] to help you backport patches. This removes
+some manual steps and is recommended.
+
+Here are the steps for the bug mentioned above:
+
+1. Install `git-node` by installing [`node-core-utils`][].
+2. Install the prerequisites for [`git-node-v8`][].
+3. Find the commit hash linked-to in the issue (in this case a51f429).
+4. Checkout a branch off the appropriate _vY.x-staging_ branch (e.g.
+   _v6.x-staging_ to fix an issue in V8 5.1).
+5. Run `git node v8 backport a51f429`.
+6. If there are conflicts, `git-node` will wait for you to resolve them.
```

Comment:
This could be clarified I think. Are we suppose to stage the changes? To commit them?